security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -12,6 +12,51 @@ These release notes track the development of {cert-manager-operator}.
 
 For more information, see xref:../../security/cert_manager_operator/index.adoc#cert-manager-operator-about[About the {cert-manager-operator}].
 
+[id="cert-manager-operator-release-notes-1-15-0"]
+== {cert-manager-operator} 1.15.0
+
+Issued: 2025-01-22
+
+Version `1.15.0` of the {cert-manager-operator} is based on the upstream cert-manager version `v1.15.4`. For more information, see the link:https://cert-manager.io/docs/releases/release-notes/release-notes-1.15/#v1154[cert-manager project release notes for v1.15.4].
+
+=== New features and enhancements
+
+*Scheduling Overrides for {cert-manager-operator}*
+
+With this release, you can configure scheduling overrides for {cert-manager-operator}, including the cert-manager controller, webhook, and CA injector.
+
+For more information, see xref:../../security/cert_manager_operator/cert-manager-customizing-api-fields.adoc#cert-manager-scheduling-overrides[Configuring Scheduling Overrides for {cert-manager-operator}].
+
+*Google CAS issuer*
+
+The {cert-manager-operator} now supports the Google Certificate Authority Service (CAS) issuer. The `google-cas-issuer` is an external issuer for cert-manager that automates certificate lifecycle management, including issuance and renewal, with CAS-managed private certificate authorities (CAs).
+
+[NOTE]
+====
+The Google CAS issuer is validated only with version 0.9.0 and {cert-manager-operator} version 1.15.0. These versions support tasks such as issuing, renewing, and managing certificates for the API server and ingress controller in {product-title} clusters.
+====
+
+*Default installMode updated to AllNamespaces*
+
+ Starting from version 1.15.0, the default and recommended Operator Lifecycle Manager (OLM) installMode is AllNamespaces. Previously, the default was SingleNamespace. This change aligns with best practices for multi-namespace operator management.
+For more information, see link:https://issues.redhat.com/browse/OCPBUGS-23406[OCPBUGS23406]
+
+=== Removed features
+
+The operator no longer includes the redundant `kube-rbac-proxy` sidecar container, reducing resource usage and complexity.
+For more information, see link:https://issues.redhat.com/browse/CM-436[CM-436]
```

Comment:
Considering that this is more of an enhancement rather than removed features, I'd suggest to move it under `New Features and Enhancements`. Also, we need to add a title for it.

WDYT @swghosh?

Reply:
++